packages/manager/modules/pci/src/projects/project/storages/containers/container/addUser/addUser.html

```diff
@@ -0,0 +1,41 @@
+<oui-modal
+    data-heading="{{ ('pci_projects_project_storages_containers_container_addUser_'+ ($ctrl.archive ? 'archive' : 'object') + '_title') | translate }}"
```

Comment:
We currently do not have an option to add user to an archive. Is `($ctrl.archive ? 'archive' : 'object')` required?